packages/block-library/src/image/edit.js

```diff
@@ -323,399 +294,359 @@ export class ImageEdit extends Component {
+		}
+	}, [ isSelected ] );
+
+	const isExternal = isExternalImage( id, url );
+	const controls = (
+		<BlockControls>
+			<BlockAlignmentToolbar
+				value={ align }
+				onChange={ updateAlignment }
+			/>
+			{ url && (
+				<MediaReplaceFlow
+					mediaId={ id }
+					mediaURL={ url }
+					allowedTypes={ ALLOWED_MEDIA_TYPES }
+					accept="image/*"
+					onSelect={ onSelectImage }
+					onSelectURL={ onSelectURL }
+					onError={ onUploadError }
+				/>
+			) }
+			{ url && (
+				<ToolbarGroup>
+					<ImageURLInputUI
+						url={ href || '' }
+						onChangeUrl={ onSetHref }
+						linkDestination={ linkDestination }
+						mediaUrl={ image && image.source_url }
+						mediaLink={ image && image.link }
+						linkTarget={ linkTarget }
+						linkClass={ linkClass }
+						rel={ rel }
+					/>
+				</ToolbarGroup>
+			) }
+		</BlockControls>
+	);
+	const src = isExternal ? url : undefined;
+	const mediaPreview = !! url && (
+		<img
+			alt={ __( 'Edit image' ) }
+			title={ __( 'Edit image' ) }
+			className={ 'edit-image-preview' }
+			src={ url }
+		/>
+	);
+
+	const mediaPlaceholder = (
+		<MediaPlaceholder
+			icon={ <BlockIcon icon={ icon } /> }
+			onSelect={ onSelectImage }
+			onSelectURL={ onSelectURL }
+			notices={ noticeUI }
+			onError={ onUploadError }
+			accept="image/*"
+			allowedTypes={ ALLOWED_MEDIA_TYPES }
+			value={ { id, src } }
+			mediaPreview={ mediaPreview }
+			disableMediaButtons={ url }
+		/>
+	);
+
+	if ( ! url ) {
+		return (
 			<>
-				<InspectorControls>
-					<PanelBody title={ __( 'Image settings' ) }>
-						<TextareaControl
-							label={ __( 'Alt text (alternative text)' ) }
-							value={ alt }
-							onChange={ this.updateAlt }
-							help={
-								<>
-									<ExternalLink href="https://www.w3.org/WAI/tutorials/images/decision-tree">
-										{ __(
-											'Describe the purpose of the image'
-										) }
-									</ExternalLink>
-									{ __(
-										'Leave empty if the image is purely decorative.'
-									) }
-								</>
-							}
-						/>
-						<ImageSizeControl
-							onChangeImage={ this.updateImage }
-							onChange={ ( value ) => setAttributes( value ) }
-							slug={ sizeSlug }
-							width={ width }
-							height={ height }
-							imageSizeOptions={ imageSizeOptions }
-							isResizable={ isResizable }
-							imageWidth={ imageWidth }
-							imageHeight={ imageHeight }
-						/>
-					</PanelBody>
-				</InspectorControls>
-				<InspectorAdvancedControls>
-					<TextControl
-						label={ __( 'Title attribute' ) }
-						value={ title || '' }
-						onChange={ this.onSetTitle }
+				{ controls }
+				<Block.div>{ mediaPlaceholder }</Block.div>
+			</>
+		);
+	}
+
+	const classes = classnames( className, {
+		'is-transient': isBlobURL( url ),
+		'is-resized': !! width || !! height,
+		'is-focused': isSelected,
+		[ `size-${ sizeSlug }` ]: sizeSlug,
+	} );
+
+	const isResizable =
+		[ 'wide', 'full' ].indexOf( align ) === -1 && isLargeViewport;
```

Comment:
Perhaps the `[ 'wide', 'full' ]` array (which is duplicated elsewhere) should be moved into a constant (e.g. WIDE_ALIGNMENTS).

Also, you could use `[].includes` here:
```suggestion
		! [ 'wide', 'full' ].includes( align ) && isLargeViewport;
```

Reply:
I used Lodash's `includes` as we do elsewhere. Not sure I see much value in `WIDE_ALIGNMENTS` unless it's something provided by the block editor itself. Maybe we should look into that.